distribution/openrct2.d.ts

```diff
@@ -2606,6 +2606,51 @@ declare global {
         "waiting_to_depart" |
         "waiting_to_start";
 
+    type CrashParticleType =  0 | 1 | 2 | 3 | 4;
+
+    /**
+     * Represents a vehicle explosion particle. They are emitted during a vehicle
+     * crash and will bounce until their timer expires and they are automatically
+     * removed.
+     */
+    interface CrashedVehicleParticle extends Entity {
+        /**
+         * The colour of the particle. Only body and trim are used.
+         */
+        colours: VehicleColour;
+
+        /**
+         * The lifetime of the particle in ticks. Default value 65535. Entity is
+         * automatically removed at 0.
+         */
+        timeToLive: number;
```

Comment:
Would it be possible to make it live forever or is that too out of scope for this PR?

Reply:
65535 ticks is 27 minutes, so not exactly forever. That would be for another PR.